exercises/practice/book-store/book_store_test.py

```diff
@@ -10,91 +10,91 @@
 class BookStoreTest(unittest.TestCase):
     def test_only_a_single_book(self):
         basket = [1]
-        self.assertEqual(total(basket), 800)
+        self.assertEqual(total(basket), 8.00)
```

Comment:
The price of a book is 800 cents, or 8 dollars.  By reducing the you are reducing the price of the books to 8 cents.

Reply:
The base unit being one cent is never specified and is not present in the C# track, for instance. The price there is treated as 8.00, and, likewise, so it is in the README. From the example:
`5 * (8 - 2.00) == 5 * 6.00 == $30.00`
Were it as you said, it would have to all be written with 800 (cents) in mind, which it isn't.

Reply:
Fair enough.  It is likely a choice in implementation as to which base unit to use.  Financials are often done with either cents or a tenth of a cent as a base unit.

Do you have an idea for improving the documentation rather than changing the tests?

Reply:
We might look at the problem-specifications repository as well to determine if the documentation should be changed there as well, if it is not specified and not present in the other tracks.

This exercise has existed since 2017, so there is a bit of history with it.